deploy/helm/templates/rbac/cluster_pod_required_role.yaml

```diff
@@ -0,0 +1,68 @@
+# permissions for end users to edit clusters.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubeblocks-cluster-pod-role
+  labels:
+    {{- include "kubeblocks.labels" . | nindent 4 }}
+rules:
+- apiGroups:
```

Comment:
missing rules of node API required to connect to kubelet?

Reply:
@leon-inf will add this later.